src/snap/config.cc

```diff
@@ -27,62 +27,62 @@ ConfigPolicy::ConfigPolicy() {}
 ConfigPolicy::~ConfigPolicy() {}
 
 ConfigPolicy::ConfigPolicy(const StringRule& rule) {
-  add_rule({""}, rule);
+    add_rule({""}, rule);
 }
 ConfigPolicy::ConfigPolicy(const IntRule& rule) {
-  add_rule({""}, rule);
+    add_rule({""}, rule);
 }
 ConfigPolicy::ConfigPolicy(const BoolRule& rule) {
-  add_rule({""}, rule);
+    add_rule({""}, rule);
 }
 
 void ConfigPolicy::add_rule(const std::vector<std::string>& ns,
                             const StringRule& rule) {
-  std::string key = build_key(ns);
-  auto policy_ptr = mutable_string_policy();
-  (*policy_ptr)[key] = rule;
+    std::string key = build_key(ns);
+    auto policy_ptr = mutable_string_policy();
+    (*policy_ptr)[key] = rule;
 }
 
 void ConfigPolicy::add_rule(const std::vector<std::string>& ns,
                             const IntRule& rule) {
-  std::string key = build_key(ns);
-  auto policy_ptr = mutable_integer_policy();
-  (*policy_ptr)[key]= rule;
+    std::string key = build_key(ns);
+    auto policy_ptr = mutable_integer_policy();
+    (*policy_ptr)[key]= rule;
 }
 
 void ConfigPolicy::add_rule(const std::vector<std::string>& ns,
                             const BoolRule& rule) {
-  std::string key = build_key(ns);
-  auto policy_ptr = mutable_bool_policy();
-  (*policy_ptr)[key] = rule;
+    std::string key = build_key(ns);
+    auto policy_ptr = mutable_bool_policy();
+    (*policy_ptr)[key] = rule;
 }
 
 Config::Config(const rpc::ConfigMap& config) : rpc_map(config) {}
 
 Config::~Config() {}
 
 bool Config::get_bool(const std::string& key) const {
-  auto bool_map = rpc_map.boolmap();
-  return bool_map.at(key);
+    auto bool_map = rpc_map.boolmap();
+    return bool_map.at(key);
 }
 
 int Config::get_int(const std::string& key) const {
-  auto int_map = rpc_map.intmap();
-  return int_map.at(key);
+    auto int_map = rpc_map.intmap();
+    return int_map.at(key);
 }
 
 std::string Config::get_string(const std::string& key) const {
-  auto str_map = rpc_map.stringmap();
-  return str_map.at(key);
+    auto str_map = rpc_map.stringmap();
```

Comment:
Thanks for the feedback! These changes fall outside of the scope of this PR, so I created [issue 49](https://github.com/intelsdi-x/snap-plugin-lib-cpp/issues/49] to ensure we address them.